Doc/library/netrc.rst

```diff
@@ -32,6 +33,10 @@ the Unix :program:`ftp` program and other FTP clients.
 
    .. versionchanged:: 3.4 Added the POSIX permission check.
 
+   .. versionchanged:: 3.7
+      Now uses :func:`os.path.expanduser` to determine the location
+      of ``~/.netrc`` if *file* is not passed as argument.
```

Comment:
netrc itself is rather UNIXy ... I assume Windows users will to use some UNIXy thing will not be bothered by a `~/` syntax, but I may be wrong.